changelog.d/20240411_000812_krishavrajsingh_CountAnnotations.md

```diff
@@ -0,0 +1,4 @@
+### Added
+
+- Added cypress test for #4662
+  (<https://github.com/cvat-ai/cvat/pull/7750>)
```

Comment:
Nothing was changed for users, so, we do not need changelog entry about adding a test